### PR TITLE
VIMC-4584: Remove separate worker image & add go-signal to worker

### PR DESCRIPTION
**Pull request by r-ash**
This PR will
* Remove separate worker image - deployments will now deploy the orderly.server image with a different endpoint
* Added go-signal to worker

### CODE REVIEW

File: R/main.R [hunk 1]
```diff
@@ -34,14 +34,19 @@ Options:
 
 main_worker_args <- function(args = commandArgs(TRUE)) {
   usage <- "Usage:
-orderly_worker [<queue_id>]"
+orderly_worker [options] [<queue_id>]
+
+Options:
+  --go-signal=PATH  Absolute path for go signal"
```
Comment (richfitz) [inline R/main.R:40]: Server line 54 implies this can be relative too (so just "path for go signal")

File: R/main.R [hunk 2]
```diff
@@ -34,14 +34,19 @@ Options:
...
   dat <- docopt_parse(usage, args)
-  list(queue_id = dat$queue_id)
+  list(queue_id = dat[["queue_id"]],
+       go_signal = dat[["go_signal"]])
 }
 
 main_worker <- function(args = commandArgs(TRUE)) {
   # nocov start
   args <- main_worker_args(args)
+  wait_for_go_signal(path, args$go_signal)
```
Comment (richfitz) [inline R/main.R:49]: If --go-signal is not supplied we start immediately, right?

**Review by richfitz — APPROVED** [review]
LGTM, I think that the go signal path can be relative, but that's a small point; update if you think it's worth it otherwise please merge

File: R/main.R (hunk 2 above)
Comment (r-ash) [reply R/main.R:49]: Yeah that's right

File: R/main.R (hunk 1 above)
Comment (r-ash) [reply R/main.R]: Thanks, updated that

**Pull request merged**